ansible/roles/biocache-properties/tasks/main.yml

```diff
@@ -20,14 +20,12 @@
 
 - name: Download SDS layers
   get_url: url={{ sds_layers_url | default('http://biocache.ala.org.au/archives/layers/sds-layers.tgz') }} dest="{{data_dir}}/biocache/layers/sds-layers.tgz" force=yes
-  when: sds_enabled is defined and sds_enabled == true
```

Comment:
@ansell Actually it's needed even if the sds_enabled is False otherwise when you ingest a dataset it will throw exceptions. I think it's bug in biocache-store that it looks for the sds-layers while sds is not enabled.